README.md

```diff
@@ -66,6 +66,7 @@ func ExampleWriteToFile() {
 	metrics := gometer.New()
 	metrics.SetFormatter(gometer.NewFormatter("\n"))
 
+	// write metrics to file periodically.
```

Comment:
What's the purpose of this comment? We have a description section for it.

Reply:
I just copied it from ./_examples/write_to_file_test.go. So I'd better remove it from here and from there.

Reply:
Oops, sorry, I thought that we don't have it.